package.json

```diff
@@ -24,6 +27,7 @@
     "bower": "~1.3.2",
     "chalk": "^0.5.1",
     "ember-cli": "^1.13.13",
+    "ember-cli-blueprint-test-helpers": "ember-cli/ember-cli-blueprint-test-helpers#qunit",
```

Comment:
I'd like to land the changes from ember-cli/ember-cli-blueprint-test-helpers#qunit in a release version before merging this PR.